engine/src/main/java/org/terasology/engine/subsystem/headless/mode/StateHeadlessSetup.java

```diff
@@ -104,6 +105,14 @@ public void init(GameEngine gameEngine) {
 
         componentSystemManager.initialise();
 
+        // Get the server to start broadcast
+        BroadcastServer broadcastServer = new BroadcastServer();
+        try {
+            broadcastServer.startBroadcast();
+        } catch (Exception e) {
```

Comment:
Is it necessary?

Reply:
It has been quite some time since I worked on this so I am not too sure, but I dont think it is necessary, will remove it.